data-reporting-service/organization-service/src/main/java/gov/cdc/etldatapipeline/organization/service/OrganizationService.java

```diff
@@ -0,0 +1,81 @@
+package gov.cdc.etldatapipeline.organization.service;
+
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrganizationElasticSearch;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrganizationKey;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrganizationReporting;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrganizationSp;
+import gov.cdc.etldatapipeline.organization.model.odse.Organization;
+import gov.cdc.etldatapipeline.organization.repository.OrgRepository;
+import gov.cdc.etldatapipeline.organization.utils.StreamsSerdes;
+import gov.cdc.etldatapipeline.organization.utils.UtilHelper;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.StreamsBuilder;
+import org.apache.kafka.streams.kstream.Consumed;
+import org.apache.kafka.streams.kstream.KStream;
+import org.apache.kafka.streams.kstream.Produced;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+
+@Service
+@RequiredArgsConstructor
+@Setter
+@Slf4j
+public class OrganizationService {
+    @Value("#{kafkaConfig.getOrganizationTopic()}")
+    private String orgTopicName;
+    @Value("#{kafkaConfig.getOrganizationElasticSearchTopic()}")
+    private String orgElasticSearchTopic;
+    @Value("#{kafkaConfig.getOrganizationReportingTopic()}")
+    private String orgReportingOutputTopic;
+
+    private final OrgRepository orgRepository;
+
+    @Autowired
+    public void processMessage(StreamsBuilder streamsBuilder) {
+
+        UtilHelper utilHelper = UtilHelper.getInstance();
+        KStream<String, Set<OrganizationSp>> organizationKStream
+                = streamsBuilder.stream(orgTopicName, Consumed.with(Serdes.String(), Serdes.String()))
+                .map((k, v) -> new KeyValue<>(
```

Comment:
can we not use single letter here? instead expand it out please

Reply:
yes.